### PR TITLE
Babel compile and new validator

### PR DESCRIPTION
**Pull request by jonasfj**
Moving to babel-compile and node 5.x

Do you see anything particularly bad?

### CODE REVIEW

File: test/runtests.sh [hunk 1]
```diff
@@ -2,27 +2,27 @@
 # USAGE: Run this file using `npm test` (must run from repository root)
```
Comment (imbstack) [inline test/runtests.sh]: While we're at it, can we get rid of this and replace it in `package.json` with `test: mocha .test/*_test.js`?

**Comment by imbstack** [comment]
r+

Hooray for spring cleaning!

**Comment by imbstack** [comment]
Oh wait, isn't this removing `npm-shrinkwrap.json`? We don't really want to do that, right?

**Comment by jonasfj** [comment]
Yeah, I'm upgrading modules.. so shrinkwrap will work again..

**Comment by jonasfj** [comment]
@imbstack,
I just pilled on... this PR now includes new validator... Have look...

**Comment by imbstack** [comment]
This looks good to me! r+

**Pull request merged**